libraries/AP_ADC/AP_ADC_ADS1115.cpp

```diff
@@ -121,9 +121,9 @@ AP_ADC_ADS1115::~AP_ADC_ADS1115()
     delete[] _samples;
 }
 
-bool AP_ADC_ADS1115::init()
+bool AP_ADC_ADS1115::init(uint8_t bus)
```

Comment:
Can you overload this function such that a call to init() with no argument will just use a default i2c bus? For example:
```
bool AP_ADC_ADS1115::init()
{
    return init(ADS1115_I2C_BUS);
}
```
See my comment regarding the default ADS1115_I2C_BUS.